src/h3lib/lib/localij.c

```diff
@@ -188,8 +188,8 @@ int h3ToLocalIjk(H3Index origin, H3Index h3, CoordIJK* out) {
             int originLeadingDigit = _h3LeadingNonZeroDigit(origin);
 
             if (FAILED_DIRECTIONS[originLeadingDigit][dir]) {
-                // TODO this part of the pentagon might not be unfolded
-                // correctly.
+                // TODO In this case this part of the pentagon might not be
```

Comment:
Nit: `In this case this part...` -> `In case this part...`

Reply:
No, I think it's correct as is - "in case" means you don't know whether something might occur, but you want to anticipate it; "in this case" means you are clear that in the current context something will occur, and you are handling it.

Reply:
But we don't know if it's actually unfolding incorrectly or not, so in case that would happen we're doing something. The current phrasing is very confusing.

Reply:
I requested this change, because every time I see this comment I think that `return 3` is thought to be possibly incorrect, instead of being a correct guard against possible issues. Open to other options. Maybe 
```
TODO: We may be unfolding the pentagon incorrectly in this case; return an error
code until this is guaranteed to be correct.
```
?

Reply:
That one is definitely clear.

Reply:
I'll update with the revised wording @nrabinowitz suggested